vumi/persist/fake_redis.py

```diff
@@ -513,6 +514,21 @@ def persist(self, key):
             return 1
         return 0
 
+    # HyperLogLog operations
+
+    @maybe_async
+    def pfadd(self, key, *values):
+        hll = self._setdefault_key(key, HyperLogLog(0.01))
```

Comment:
Redis' quoted accuracy is `0.81%`. I'm guessing `0.01` is `1%`? Does that mean we should be using `0.0081` or `0.01`?